imi/src/main/java/org/motechproject/nms/imi/service/impl/CdrFileServiceImpl.java

```diff
@@ -724,6 +731,8 @@ public void verifyDetailFileChecksumAndCount(CdrFileNotificationRequest request)
                 null
         ));
 
+        // TODO: is there a better place/way to trigger this?
+        cleanOldCallRecords();
```

Comment:
The biggest concern here was that I didn't want to touch call records if there was any chance of existing cdrs and csrs being processed... causing more issues with paging? Since there wasn't a way to figure out that cdr/csr processing is completely done(true?), I figured we can run the cleanup job before targetfile generation every morning?